extensions/amp-apester-media/0.1/monetization/companion/display.js

```diff
@@ -84,6 +85,10 @@ function constructCompanionDisplayAd(slot, bannerSizes, apesterElement) {
   );
   ampAd.classList.add('amp-apester-companion');
   apesterElement.parentNode.insertBefore(ampAd, apesterElement.nextSibling);
-  apesterElement.getResources().attemptChangeSize(ampAd, maxHeight);
+  Services.mutatorForDoc(apesterElement).attemptChangeSize(
+    ampAd,
+    maxHeight,
+    undefined
```

Comment:
Hmm just don't pass the `undefined`?

Reply:
`gulp check-types` would fail if not provided. In the past it can't resolve `apesterElement.getResources()` at all so it does not see a problem.

Reply:
I see, it's because each one of the width/height is optional but not both at the same time. A little tricky, so let's keep it but just add a comment on what variable that is.

Reply:
Tx. added comments.